r2i/onnxrt/loader.cc

```diff
@@ -24,6 +21,14 @@
 namespace r2i {
 namespace onnxrt {
 
+/* Custom deleter */
+template< typename T >
```

Comment:
This is unnecessary. The correct way to use a smart pointer with arrays is to do

`array = std::make_shared<int[]>(new int[SIZE])`.

This way it'll use `delete []` rather than plain `delete`